### ISSUE TITLE
date front matter not needed in blog posts

### ISSUE THREAD

**Issue by tobyhodges** [carpentries-incubator/jekyll-pages-novice#232]
One of the magic things that Jekyll does is define `page.date` from a yyyy-mm-dd date string in the filename. (This can be over-ridden by defining a different value in the `date` field of the page front matter.)

I do not believe we should teach it this way from the start (too much magic) but it is probably worth a callout at the end of Working with Filters... What do others think?

**Comment by anenadic** [comment]
I agree with both points - too much magic at the beginning but a callout at the end would be great.
